compiler/src/dotty/tools/dotc/printing/RefinedPrinter.scala

```diff
@@ -480,6 +480,7 @@ class RefinedPrinter(_ctx: Context) extends PlainPrinter(_ctx) {
         if (lo eq hi) optText(lo)(" = " ~ _)
         else optText(lo)(" >: " ~ _) ~ optText(hi)(" <: " ~ _)
       case Bind(name, body) =>
+        ("implict ": Text).provided(tree.symbol.is(Implicit) && !ctx.settings.YtestPickler.value) ~ // Used for scala.quoted.Type in quote patterns (not pickled)
```

Comment:
1. Typo: implicit.
2. Also, what about new style implicits? Should quoted types not be flagged `Delegate` instead of `Implicit`?
3. Instead of `ctx.settings.YtestPickler.value` use `homogenizedView`